neo/app/assets/javascripts/lib/dc.leaflet.js

```diff
@@ -0,0 +1,434 @@
+dc.leafletChart = function(_chart) {
```

Comment:
Understood. Then could you add a comment to introduce where this code come from and serve what purpose? and probably check their License to see if it can be directly copied. Usually we need to include their license header in the first several lines of the file.